release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -421,6 +421,15 @@ The crun low-level container runtime is now generally available in {product-titl
 
 Linux Control Group version 2 (cgroup v2) is now generally available in {product-title} 4.13. There is no new functionality in the GA version.
 
+[id="ocp-4-13-graceful-node-shutdown"]
+==== Support for graceful node shutdown
+
+A graceful node shutdown delays the eviction of pods during a node shutdown. In {product-title} {product-version}, you can configure the kublet to enable a graceful node shutdown to ensure that pods running critical workloads are not interrupted.
+
+To configure graceful node shutdowns, specify a termination grace period for regular and critical pods in the `KubletConfig` custom resource. A termination grace period defines a time period for the pod to complete any ongoing tasks before terminating. You can also add priority classes to pods to specify the order of termination.
```

Comment:
Also need update `KubletConfig` to `KubeletConfig`

Reply:
Done!